### PR TITLE
[plclogo] Remove org.apache.commons.net.util.SubnetUtils

### PR DESCRIPTION
**Pull request by lsiepel**
- Remove org.apache.commons
 
Depends on : https://github.com/openhab/openhab-core/pull/4375

### CODE REVIEW

File: bundles/org.openhab.binding.plclogo/src/main/java/org/openhab/binding/plclogo/internal/discovery/PLCDiscoveryService.java [hunk 1]
```diff
@@ -107,44 +101,19 @@ protected void startScan() {
...
-                InetAddress address = iface.getAddress();
-                if (address instanceof Inet4Address) {
-                    String prefix = String.valueOf(iface.getNetworkPrefixLength());
-                    SubnetUtils utilities = new SubnetUtils(address.getHostAddress() + "/" + prefix);
-                    addresses.addAll(Arrays.asList(utilities.getInfo().getAllAddresses()));
-                }
-            }
-        }
-
+        List<InetAddress> addressesToScan = NetUtil.getFullRangeOfAddressesToScan();
```
Comment (lolodomo) [inline bundles/org.openhab.binding.plclogo/src/main/java/org/openhab/binding/plclogo/internal/discovery/PLCDiscoveryService.java:104]: Previous code was ignoring interfaces that were no up and also loopback interface.
The new code does not ?

Comment (lsiepel) [reply bundles/org.openhab.binding.plclogo/src/main/java/org/openhab/binding/plclogo/internal/discovery/PLCDiscoveryService.java:104]: `NetUtil.getFullRangeOfAddressesToScan()`  calls `NetUtil.getAllInterfaceAddresses()` and that method filters:

https://github.com/openhab/openhab-core/blob/389f6a34343a723442829b7d9841240adc2f2a9c/bundles/org.openhab.core/src/main/java/org/openhab/core/net/NetUtil.java#L344

Comment (lolodomo) [reply bundles/org.openhab.binding.plclogo/src/main/java/org/openhab/binding/plclogo/internal/discovery/PLCDiscoveryService.java:104]: Fine

**Review by lolodomo — APPROVED** [review]
LGTM, thank you

**Pull request merged**